src/resource/resource.test.ts

```diff
@@ -701,3 +708,68 @@ describe("cloneResource", () => {
     expect(clonedObject).not.toBe(sourceObject);
   });
 });
+
+describe("getContainedResourceUrlAll", () => {
+  const initContainer = (
+    containerUrl: string,
+    containedResourceUrls: UrlString[]
+  ) => {
+    let childrenIndex = createThing({ url: containerUrl });
+    childrenIndex = addUrl(childrenIndex, rdf.type, ldp.BasicContainer);
+
+    let mockContainer = mockContainerFrom(containerUrl);
+
+    containedResourceUrls.forEach((thingName) => {
+      let childListing = createThing({
+        url: containerUrl + thingName + ".ttl",
+      });
+      childListing = addUrl(childListing, rdf.type, ldp.Resource);
+      childListing = addDatetime(childListing, dct.modified, new Date());
+
+      childrenIndex = addUrl(childrenIndex, ldp.contains, childListing);
+      mockContainer = setThing(mockContainer, childListing);
+    });
+
+    mockContainer = setThing(mockContainer, childrenIndex);
+    childrenIndex = addUrl(childrenIndex, rdf.type, ldp.Container);
+
+    return mockContainer;
+  };
+
+  it("gets all urls for contained resources from a container", () => {
+    const containerUrl = "https://arbitrary.pod/container/";
+    const containedThings = ["resource1", "resource2", "resource3"];
+    const container = initContainer(containerUrl, containedThings);
+    const expectedReturnUrls = containedThings.map(
+      (thingName) => `${containerUrl}${thingName}.ttl`
+    );
+
+    expect(getContainedResourceUrlAll(container)).toStrictEqual(
```

Comment:
Heh, I didn't know about `toStrictEqual`, but looks like that makes more sense to use in most cases where we use `toEqual`. TIL!